<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderProgressWidget</class>
 <widget class="QWidget" name="OrderProgressWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>459</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Order Progress</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Order status</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="EnhancedTableView" name="tableViewOrder"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>View orders</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxAcquired">
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::StrongFocus</enum>
           </property>
           <property name="text">
            <string>Acquired</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxInProgress">
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::StrongFocus</enum>
           </property>
           <property name="text">
            <string>In progress</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxProcessed">
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::StrongFocus</enum>
           </property>
           <property name="text">
            <string>Processed</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Limit to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="RelationalComboBox" name="comboBoxLimit">
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::StrongFocus</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Barcode scan</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidgetScans"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Barcode scan</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEditBarcode">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit#lineEditBarcode {
	color: black;
	background: transparent;
	border: 3px solid blue;
	border-radius: 5px;
	padding: 0 8px;
}
QLineEdit#lineEditBarcode::focus {
	color: black;
	background: cyan;
	border: 3px solid blue;
	border-radius: 5px;
	padding: 0 8px;
}</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Edit scan</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButtonSetAsUnprocessed">
              <property name="text">
               <string>Set as unprocessed</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EnhancedTableView</class>
   <extends>QTableView</extends>
   <header>App.Widget.View</header>
  </customwidget>
  <customwidget>
   <class>RelationalComboBox</class>
   <extends>QComboBox</extends>
   <header>App.Widget.Control</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableViewOrder</tabstop>
  <tabstop>checkBoxAcquired</tabstop>
  <tabstop>checkBoxInProgress</tabstop>
  <tabstop>checkBoxProcessed</tabstop>
  <tabstop>comboBoxLimit</tabstop>
  <tabstop>tableWidgetScans</tabstop>
  <tabstop>lineEditBarcode</tabstop>
  <tabstop>pushButtonSetAsUnprocessed</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
